core/trino-main/src/test/java/io/trino/server/TestDynamicFilterService.java

```diff
@@ -133,7 +132,9 @@ public void testDynamicFilterSummaryCompletion()
         assertEquals(stats.getReplicatedDynamicFilters(), 0);
         assertEquals(
                 stats.getDynamicFilterDomainStats(),
-                ImmutableList.of(new DynamicFilterDomainStats(filterId, getExpectedDomainString(1L, 3L), 3, 0)));
+                ImmutableList.of(new DynamicFilterDomainStats(
+                        filterId,
+                        getExpectedDomainStatsString(1L, 3L, 3, INTEGER))));
```

Comment:
I think we know simplified domain here. Could we generate it and compare `toStrings`?